plugins/catalog-backend/src/processors/UrlReaderProcessor.test.ts

```diff
@@ -31,6 +31,8 @@ import {
 import { defaultEntityDataParser } from '../util/parse';
 import { UrlReaderProcessor } from './UrlReaderProcessor';
 import { UrlReaders } from '@backstage/backend-defaults/urlReader';
+import { UrlReaderService } from '@backstage/backend-plugin-api';
+import { mockApis } from '@backstage/test-utils';
```

Comment:
Use the config mock from `mockServices` from `@backstage/backend-test-utils` instead